app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -43,20 +47,47 @@ export const ContentForm: React.FC<IContentFormProps> = ({
 }) => {
   const [{ userInfo }] = useApp();
   const [{ impersonate }] = useProfileStore();
+  const [{ tonePools }] = useLookup();
 
   if (!content) return null;
 
   const userId = impersonate?.id ?? userInfo?.id ?? 0;
   const isAV = content.contentType === ContentTypeName.AudioVideo;
-  const versions = content.versions?.[userId] ?? {
-    byline: content.byline,
-    headline: content.headline,
-    summary: '',
-    body: isAV
-      ? content.isApproved && content.body
-        ? content.body
-        : content.summary
-      : content.body,
+
+  const sentiment =
+    content.versions?.[userId]?.tone !== undefined && content.versions?.[userId]?.tone != null
+      ? content.versions?.[userId]?.tone
+      : content.tonePools.length
+      ? content.tonePools[0].value
+      : undefined;
+
+  const versions = {
+    ...content.versions?.[userId],
+    byline: content.versions?.[userId]?.byline ?? content.byline,
+    headline: content.versions?.[userId]?.headline ?? content.headline,
+    summary: content.versions?.[userId]?.summary ?? '',
+    body:
+      content.versions?.[userId]?.body ??
+      (isAV ? (content.isApproved && content.body ? content.body : content.summary) : content.body),
+    tonePools:
```

Comment:
Why are you using a property `tonePools` here?  The model is for `IContentVersionModel` has a `tone` property you added.